### ISSUE TITLE
Can't download the dogc vs non-dogs dataset

### ISSUE THREAD

**Issue by k4ntz** [megvii-research/FSSD_OoD_Detection#7]
Hi. I am trying to reproduce and use your dogs vs non-dogs dataset. I followed the link to download it but the archive file seems to be broken. 
I have read [this issue](https://github.com/megvii-research/FSSD_OoD_Detection/issues/1), where you explain how you created the dataset. 
in it, you declare using 50K images :

-  **Dogs-50A-train**: 50K images 1000 images from ImageNet Training Set for each 50 classes (I guess, because you wrote _100_). Another problem is that for certain classes, you only have ~700 images.
- **Dogs-50A-val**: 10K images (so I guess 200 images for each 50 classes). You write once that you take it from ImageNet validation (in the table) and then from the training set. The validation set only contain 50 images per class. I thus think that you might have taken these images from the training set. Am I right? 

Would you have a link to a non-broken archive ? Or could you please clear my above concerns such that I can generate an equivalent one ?

**Comment by romain-xu-darme** [comment]
Hi. I'm facing the same issue right now. Did you ever get a link to non-broken files? @andrehuang  @PKUCSS Not sure if it helps but here are the checksum of the available files
md5sum dogs50A-train.tar.gz 859653679d267b902666c7d629a44a0d
md5sum non-dogs-val.tar.gz 62d737cf91c7909f3d7796e701817a40

**Comment by andrehuang** [comment]
Hi, sorry for the late response. Thanks for your interest in our work. The authors of this paper have either left the company or graduated, so it's not watched very promptly. 
For the file, it's a bit weird that it cannot be unzipped. But I'm not sure whether we have another copy offline now. @PKUCSS  Sishuo, do you have one?
Also, for the creation of the dataset, Sishuo, can you clarify a bit? Thanks!

**Comment by PKUCSS** [comment]
Sorry that I don't store a local copy of the dataset or the code for construction after I finished my internship at Megvii, so currently I'm not able to check the details. We will try to recover the dataset and clarify the details if a local copy can be found.

**Comment by romain-xu-darme** [comment]
Ok, thanks a lot!
